nuxt.config.js

```diff
@@ -101,7 +101,7 @@ export default {
       {
         config: {
           /* eslint-disable-next-line no-undef */
-          apiKey: process.env.FIREBASE_API_KEY,
+          apiKey: process.env.LASHAWN_FIREBASE_API_KEY,
```

Comment:
Did you specifically want to have your name in the public repo here? (np if so, just wanted to check)

Reply:
Nope! I didn't mean to check that in. Thank you for catching it!